framework/src/org/checkerframework/framework/util/defaults/QualifierDefaults.java

```diff
@@ -119,6 +119,22 @@
     };
 
     /**
+     * Standard unchecked default locations that should be top
+     */
+    // Fields are default to top so that warnings are issued at field reads, which we believe are more common
+    // than field writes. Future work is to specify different defaults for field reads and field writes.
```

Comment:
No, it is also possible that the bytecode assumes that the field is something other than top and uses it as such.  For example, bytecode could assume a filed is non-null.  So writing null to that field could cause a null pointer exception.